Makefile

```diff
@@ -81,3 +90,7 @@ check-code:
 	yarn tsc-check
 	yarn eslint-check
 	yarn build
+
+create-tmp-env-file:
```

Comment:
I think this is only ever used in `ci` - perhaps we can reflect that in the name.